Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.14.4-alpine as builder
+FROM golang:1.17.3-alpine3.14 as builder
 LABEL maintainer "George Tankersley <george@zfnd.org>"
```

We might want to update the maintainer while we're tweaking things:
```suggestion
LABEL maintainer "Zcash Foundation <engineers@zfnd.org>"
```